### PR TITLE
feat(endpoint): added a local health endpoint

### PR DESCRIPTION
**Pull request by turkeychikken**
Solves #950 

haven't updated docs or added tests, thoughts?

### CODE REVIEW

**Comment by andrewazores** [comment]
This pretty much looks like what I expected. I would say use the `204` status code and an empty response body, just for simplicity's sake. Rather than the path `/local`, which may not be immediately clear to an API client what that means, maybe use `/health/liveness`?

**Comment by andrewazores** [comment]
Looks good, just update `HTTP_API.md` and add some basic unit tests please.

File: src/main/java/io/cryostat/net/web/http/generic/LocalHealthGetHandler.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import javax.inject.Inject;
+
+import io.cryostat.net.security.ResourceAction;
+import io.cryostat.net.web.http.RequestHandler;
+import io.cryostat.net.web.http.api.ApiVersion;
+
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.RoutingContext;
+
+class LocalHealthGetHandler implements RequestHandler {
```
Comment (andrewazores) [inline src/main/java/io/cryostat/net/web/http/generic/LocalHealthGetHandler.java:51]: Rename this class please. Generally the class name follows the endpoint path and the HTTP verb, so in this case I would suggest "HealthLivenessGetHandler".

Comment (turkeychikken) [reply src/main/java/io/cryostat/net/web/http/generic/LocalHealthGetHandler.java]: Okay done.

Comment (andrewazores) [reply src/main/java/io/cryostat/net/web/http/generic/LocalHealthGetHandler.java]: Nope, not a problem at all. This is normal and expected :-)

**Review by andrewazores — APPROVED** [review]

**Pull request merged**